### PR TITLE
docs: brings back docs

### PR DESCRIPTION
**Pull request by jalavik**
Brings back docs from v2 branches.

To build do: `sphinx-build -qnNW docs docs/_build/html`.

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -91,8 +91,3 @@ Access the record (web/rest)
 
    firefox http://localhost:5000/records/1
    curl -i -H "Accept: application/json" http://localhost:5000/api/records/1
-
-
-======
-ENJOY!
-======
```
Comment (kaplun) [inline INSTALL.rst]: No!? We shall be sad?

File: docs/grobid.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+The models available after cloning are not using the new available training
+data. To generate the new ones we need to go inside of the root folder and
+call:
+
+.. code-block:: bash
+
+    $ cd grobid
+    $ java -Xmx1024m -jar grobid-trainer/target/grobid-trainer-0.3.4-SNAPSHOT.one-jar.jar 0 $MODEL -gH grobid-home
+
+where ``$MODEL`` is the model we want to train. Note that there's new data
```
Comment (kaplun) [inline docs/grobid.rst]: Is there more information on what is a `$MODEL`? Can you add some online pointer?

Comment (jacquerie) [inline docs/grobid.rst]: That would be http://grobid.readthedocs.io/en/latest/Training-the-models-of-Grobid/, but in fact the only models that makes sense to train are the ones listed below (`segmentation` and `header`).

Comment (kaplun) [inline docs/grobid.rst]: I miss something: "should I put really the word `segmentation` there? And where the input is provided?

Comment (jacquerie) [inline docs/grobid.rst]: Yes, you should put the word `segmentation` there. Training data (the input) MUST be in folder `grobid-trainer/resources/dataset/$MODEL/corpus`.

Comment (jalavik) [inline docs/grobid.rst]: So shall I update something here or are we okay to 🚢  for now?

File: INSTALL.rst (hunk 1 above)
Comment (jalavik) [inline INSTALL.rst]: Hehe, I brought it back, but without the headline causing issues with table of contents.

File: docs/grobid.rst (hunk 2 above)
Comment (jacquerie) [inline docs/grobid.rst]: For me, :shipit:, but let's hear what @kaplun says.

Comment (kaplun) [inline docs/grobid.rst]: Ah no please go on! We can always fix it later.

Comment (kaplun) [inline docs/grobid.rst]: Is there just a reference that can be put (E.g. to official grobid documentation) to exaplain how the corpus should be structured then?

**Pull request merged**